inputs_supply.md

```diff
@@ -203,7 +203,7 @@ The costs associated with land, irrigation, and fertilizer are endogenously dete
 
 ##### Emissions 
 
-Historical non-CO<sub>2</sub> emissions information is provided in several files within the [CEDS folder]((https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/emissions/CEDS).
+Historical non-CO<sub>2</sub> emissions information is provided in the GCAM release as "pre-built" data aggregated to GCAM regions, technologies, and fuels. Users that want to build using CEDS raw data, for example to build for different regional aggregations, will need to generate CEDS data using the open-source [CEDS system](https://github.com/JGCRI/CEDS) and place the resulting emissions data by country, fuel, and sector within the [CEDS folder](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/emissions/CEDS).
```

Comment:
Is this the right place to add the CEDS version number? Its [CEDS v_2021_04_21 ](https://github.com/JGCRI/CEDS).